src/components/ui/productCard/ProductCard.tsx

```diff
@@ -1,19 +1,33 @@
 import React from "react";
+import { Link } from "react-router-dom";
 import "./productCard.scss";
 import { ProductCardProps } from "../../../types";
 
 export default function ProductCard(props: ProductCardProps) {
   return (
     <div
       style={props.limit < props.dogy.id ? { display: "none" } : undefined}
-      className="product-сard container-fluid"
+      className="product-сard"
     >
-      <div className="product-сard__img-cuter">
-        <img alt={`>>>>>>>>> ${props.dogy.name}`} src={props.dogy.image.url} />
-      </div>
+      {props.dogy.image ? (
+        <img
+          className="product-сard__image"
+          alt={props.dogy.name}
+          src={props.dogy.image.url}
+        />
+      ) : (
+        <div className="product-сard__image">No image</div>
+      )}
 
       <p className="product-сard__type">{props.dogy.life_span}</p>
       <p className="product-сard__name">{props.dogy.name}</p>
+      <Link
+        to={{ pathname: `/products/${props.dogy.id}` }}
+        state={{ id: props.dogy.id }}
+        className="product-сard__link"
+      >
+        Details
+      </Link>
     </div>
```

Comment:
I think it's better to destructure props in the arguments of a function - `ProductCard({ dogy }: ProductCardProps`) 
and then use just `dogy.name`, `dogy.image.url`